front_end/core/host/RNPerfMetrics.ts

```diff
@@ -137,7 +165,15 @@ export type BrowserVisibilityChangeEvent = Readonly<{
   }>,
 }>;
 
-export type ReactNativeChromeDevToolsEvent =
-    EntrypointLoadingStartedEvent|EntrypointLoadingFinishedEvent|DebuggerReadyEvent|BrowserVisibilityChangeEvent;
+export type UnhandledErrorEvent = Readonly<{
+  eventName: 'Browser.UnhandledError',
+  params: Readonly<{
+    type: 'error' | 'rejectedPromise',
+    message: string,
+  }>,
+}>;
+
+export type ReactNativeChromeDevToolsEvent = EntrypointLoadingStartedEvent|EntrypointLoadingFinishedEvent|
+    DebuggerReadyEvent|BrowserVisibilityChangeEvent|UnhandledErrorEvent;
```

Comment:
Can this be formatted to something like...?

```
export type ReactNativeChromeDevToolsEvent =
    | EntrypointLoadingStartedEvent
    | EntrypointLoadingFinishedEvent
    | DebuggerReadyEvent
    | BrowserVisibilityChangeEvent
    | UnhandledErrorEvent;
```

Reply:
Yeah unfortunately it's the formatting used upstream.

https://github.com/facebookexperimental/rn-chrome-devtools-frontend/blob/24b0c8192bcc50e7274cdd1aea4ef6f425b77f48/front_end/core/common/Color.ts#L133-L134

https://github.com/facebookexperimental/rn-chrome-devtools-frontend/blob/24b0c8192bcc50e7274cdd1aea4ef6f425b77f48/front_end/core/sdk/ConsoleModel.ts#L820-L822

Don't think much changed since last time (https://github.com/facebookexperimental/rn-chrome-devtools-frontend/pull/27#discussion_r1535804721), but let's see if we can get some official guideline from the Chrome team for formatter/linter setup